storybook/test/__snapshots__/index.js.snap

```diff
@@ -8,7 +8,7 @@ exports[`Storyshots Components/Animate Appear Bottom Left 1`] = `
     className="components-notice__content"
   >
     <p>
-      Appear animation. Origin: 
```

Comment:
The line breaks in this paragraph are a little weird, with a period on its own line. But will the removed space not mean it says "Origin:bottom left" now? I haven't yet had time to dive deep into storybook stuff, so I could be wrong.

Reply:
These are automatically generated by Storybook's test snapshots. We can typically ignore these updates during review.

Hope this helps!